ClipboardMonitor/NativeMethods.cs

```diff
@@ -1,59 +1,16 @@
 ﻿using System;
 using System.Runtime.InteropServices;
-using System.Text;
 
 namespace ClipboardMonitor
 {
+    // These methods cannot be provided by CsWin32 properly.
```

Comment:
I see the bugs you filed on these, but as I explain on each of them, I believe CsWin32 is in fact doing it correctly, albeit perhaps not with the signature you prefer.